client/post-editor/edit-post-status/index.jsx

```diff
@@ -1,48 +1,53 @@
 /**
  * External dependencies
  */
-const React = require( 'react' ),
-	noop = require( 'lodash/noop' );
+import React, { PropTypes, Component } from 'react';
+import noop from 'lodash/noop';
+import { moment, translate } from 'i18n-calypso';
```

Comment:
And while it's included in the docs there, the reason for not calling on them directly is:

>The advantage of using a higher-order component instead of calling translate directly from the `i18n-calypso` module is that the latter does not properly account for change events which may be emitted by the state emitter object.